.gitignore

```diff
@@ -1,4 +1,5 @@
 *.pyc
 virtenv/
 venv/
+virtual/
```

Comment:
@iamkatrinaanne virtualenv folder is already ignored (venv and virtenv) please be consistent on virtualenv naming so we don't need to add new item to the gitignore file for every virtualenv names